topics/linux/linux_exercise.md

```diff
@@ -119,6 +119,17 @@ Jetzt möchten wir, dass das hello-world Skript beim Aufstarten automatisch gest
 und die Meldungen ins journald schreibt.  
 Nutze für diese Übung vim.
 
+In Github Codespaces steht nicht direkt systemd zur Verfügung.
+Deshalb ist in OCI image vorbereitet, in dem wir systemd zur Verfügung haben.
```

Comment:
ich würde hier "Container Image" schreiben. OCI ist vermutlich nicht allen bekannt. Wird erst in späterem Modul behandelt.

Reply:
Container Image ist bisschen komisch, weil man genau die beiden konzepte trennen sollte mental.
Ich ändere es, kann man ja wieder zurück ändern.